### ISSUE TITLE
reMarkable build 3.6.0.1806 changes

### ISSUE THREAD

**Issue by loricott** [rien/reStream#95]
I just updated my reMarkable to the release in question, and now the video stream is mirrored and apparently only a quarter of the screen is visible.
I am more than willing to help in solving this, and I would appreciate if someone could point me in the good direction.

**Comment by loricott** [comment]
![IMG_20230804_171738](https://github.com/rien/reStream/assets/40783801/f77fe480-9f9c-4c3b-84de-2976f69d0011)
![IMG_20230804_171750](https://github.com/rien/reStream/assets/40783801/6c5e009c-b8b4-4e1f-becd-e7273c624a00)

For reference.

**Comment by loricott** [comment]
If it helps with the debugging, here is what I experimented with.
I edited the reStream.sh file as follows:
1.  adding `vflip` in the video_filters options corrects the image orientation
2. in the selection of the remarkable model (lines 144+) the latest remarkable update does not have the /dev/shm/swtfb.01 entry, therefore it falls into the 'else' section. There, using pixel_format="rgb565le" and bytes_per_pixel=2 manages to get a readable image, with the drawback that the background is blue.

**Comment by mahrud** [comment]
The following worked for me:
```
            bytes_per_pixel=2
            pixel_format="gray16be"
            video_filters="$video_filters,transpose=3"
```

**Comment by jimmy9065** [comment]
> The following worked for me:
> 
> ```
>             bytes_per_pixel=2
>             pixel_format="gray16be"
>             video_filters="$video_filters,transpose=3"
> ```

This is not working for me (ver 3.5.2.1807)
in this case, have to use 
```
        else
            width=1872
            height=1404
            bytes_per_pixel=2
            fb_file=":mem:"
            pixel_format="gray16be"
            video_filters="$video_filters,transpose=2"
        fi
```
I have have 4 screen overlapped together and flashing
![image](https://github.com/rien/reStream/assets/10953593/058dcf45-ad07-4efd-aafd-261cf78fc3ab)

**Comment by jimmy9065** [comment]
this is the closet I got
```
150   ¦ ¦ ¦ ¦ ¦ width=1404
151   ¦ ¦ ¦ ¦ ¦ height=2800
152   ¦ ¦ ¦ ¦ ¦ bytes_per_pixel=2
153   ¦ ¦ ¦ ¦ ¦ fb_file=":mem:"
154   ¦ ¦ ¦ ¦ ¦ pixel_format="gray16" 
```
![image](https://github.com/rien/reStream/assets/10953593/37c95bae-3384-46ca-9c27-b578559d9cb1)

**Comment by kimtore** [comment]
On firmware version 3.8.3.197 I got problems with grey banding, wrong resolution, and mirrored image.

Thanks to the helpful messages in this thread, the following tweaks fixed it:

```
# in the top of the file
video_filters="curves=all='0/0 0.07/1 1/1'"

# [...]
case "$rm_version" in
# [...]
        else
            bytes_per_pixel=2
            pixel_format="gray16be"
            video_filters="$video_filters,transpose=3"
            width=1872
            height=1404
            fb_file=":mem:"
        fi
```

**Comment by jtfidje** [comment]
> On firmware version 3.8.3.197 I got problems with grey banding, wrong resolution, and mirrored image.
> 
> Thanks to the helpful messages in this thread, the following tweaks fixed it:
> 
> ```
> # in the top of the file
> video_filters="curves=all='0/0 0.07/1 1/1'"
> 
> # [...]
> case "$rm_version" in
> # [...]
>         else
>             bytes_per_pixel=2
>             pixel_format="gray16be"
>             video_filters="$video_filters,transpose=3"
>             width=1872
>             height=1404
>             fb_file=":mem:"
>         fi
> ```

This also worked for me today on my reMarkable 2, running version 3.8.31978

**Comment by itay-grudev** [comment]
To address the dark image issue, add an `eq=gamma=10` filter, like so:

```
            video_filters="$video_filters,transpose=3,eq=gamma=10"
```

**Comment by rien** [comment]
This should be fixed in the latest release by #100. Let me know if you encounter any additional issues.
